components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  padding-inline: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
+  
   margin: 0;
+  text-decoration: none;
+}
 
-  border-inline-start: var(--spectrum-listitem-texticon-focus-indicator-size) solid
-    transparent;
+.spectrum-Menu-item {
+  display: grid;
+  grid-template-areas:
+    "selectedArea chevronAreaCollapsible checkmarkArea iconArea labelArea        valueArea actionsArea chevronAreaDrillIn"
+    ".            .                      .             .        descriptionArea  .         .           .                 "
+    ".            .                      .             .        submenuArea      .         .           .                 ";
+  
+  grid-template-columns: auto auto auto auto 1fr auto auto auto;
+  grid-template-rows: 1fr auto;
+}
 
-  min-block-size: var(--spectrum-listitem-texticon-height);
+.spectrum-Menu-item .spectrum-Menu-item {
+  display: grid;
+  grid-template-areas: "submenuItemLabelArea submenuItemValueArea";
+  grid-template-columns: auto auto;
+  grid-template-rows: 1fr auto;
+}
 
-  font-size: var(--spectrum-listitem-texticon-text-size);
-  font-weight: var(--spectrum-listitem-texticon-text-font-weight);
-  font-style: normal;
-  text-decoration: none;
+.spectrum-Menu-item .spectrum-Menu-item .spectrum-Menu-itemLabel {
+  grid-area: submenuItemLabelArea;
+}
+
+.spectrum-Menu-item .spectrum-Menu-item .spectrum-Menu-itemValue {
+  grid-area: submenuItemValueArea;
+}
+
+.spectrum-Menu-item:focus {
+  outline: none;
+
+  .spectrum-Menu-itemLabel {
+    color: var(--highcontrast-menu-item-label-content-color-focus, var(--mod-menu-item-label-content-color-focus, var(--spectrum-menu-item-label-content-color-focus)));
+  }
+
+  .spectrum-Menu-sectionHeading {
+    color: var(--highcontrast-menu-section-header-color, var(--mod-menu-section-header-color, var(--spectrum-menu-section-header-color)));
+  }
+
+  .spectrum-Menu-itemDescription {
+    color: var(--highcontrast-menu-item-description-color-focus, var(--mod-menu-item-description-color-focus, var(--spectrum-menu-item-description-color-focus)));
+  }
+
+  .spectrum-Menu-itemValue {
+    color: var(--highcontrast-menu-item-value-color-focus, var(--mod-menu-item-value-color-focus, var(--spectrum-menu-item-value-color-focus)));
+  }
+  
+  .spectrum-Menu-itemIcon:not(.spectrum-Menu-chevron, .spectrum-Menu-checkmark) {
+    fill: var(--highcontrast-menu-item-label-icon-color-focus, var(--mod-menu-item-label-icon-color-focus, var(--spectrum-menu-item-label-icon-color-focus)));
+  }
+
+  .spectrum-Menu-chevron {
+    fill: var(--highcontrast-menu-collapsible-icon-color, var(--mod-menu-collapsible-icon-color, var(--spectrum-menu-collapsible-icon-color)));
+  }
+
+  .spectrum-Menu-checkmark {
+    fill: var(--highcontrast-menu-checkmark-icon-color-focus, var(--mod-menu-checkmark-icon-color-focus, var(--spectrum-menu-checkmark-icon-color-focus)));
+  }
+}
+
+.spectrum-Menu-item:focus-ring,
```

Comment:
:focus-ring  is depreciated, it's not giving the 10% black background color like it should:
<img width="159" alt="menu item with focus-ring" src="https://user-images.githubusercontent.com/25614178/192045381-2e8d5f82-d666-4492-964d-5d915b265dc5.png">
_[fig 1]_

we can use `.spectrum-Menu-item:focus-visible` which gives me this (10% black background)
In Chrome:
<img width="172" alt="menu item with focus-visible" src="https://user-images.githubusercontent.com/25614178/192045549-5bcc1079-7ac6-48d4-b8fc-e9aa040286e3.png">
_[fig 2]_

In Firefox:
<img width="162" alt="Screen Shot 2022-09-23 at 4 04 49 PM" src="https://user-images.githubusercontent.com/25614178/192048435-8cb66ba8-33b0-4882-ad6a-70c007d15ab5.png">
_[fig 3]_

The background is correct with :focus-visible, but the result does not match the design as shown at https://spectrum.adobe.com/page/menu/
<img width="400" alt="menu design" src="https://user-images.githubusercontent.com/25614178/192043890-b8944244-e452-40bd-a177-2513ad897a3e.png">
_[fig 4]_

The incorrect blue outline we are seeing in fig2 and fig3 is the result of the outline being removed from the :focus state on line 389 but not from the :focus-visible state.

Reply:
Woah, wild. I did not see this happening when I worked on it. I've made some changes to this focus state, so hopefully this is fixed now.